Explorer/Assets/DCL/Input/Component/InputMapComponent.cs

```diff
@@ -42,6 +44,37 @@ public InputMapComponent(Kind flags)
         {
             active = flags;
             IsDirty = true;
+            inputBlockCounters = new Dictionary<Kind, int>(VALUES.Count);
+
+            for (var i = 0; i < VALUES.Count; i++)
+            {
+                Kind kind = VALUES[i];
+                inputBlockCounters.Add(kind, EnumUtils.HasFlag(active, kind) ? 0 : 1);
+                ReportHub.LogError(ReportData.UNSPECIFIED, $"Added BlockCounter for {kind.ToString()} value {inputBlockCounters![kind]}");
```

Comment:
why it is error? and if so, can you describe it better inside the message

Reply:
AH its not error!! Was a log I was printing to validate stuff and this one escaped the cleanse!!